tests/scripts/depends-ciphers.pl

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env perl
+
+# depends-ciphers.pl
+#
+# Copyright (c) 2018, ARM Limited, All Rights Reserved
+#
+# Purpose
+#
+# To test the code dependencies on individual ciphers in each test suite. This
```

Comment:
As with the original https://github.com/ARMmbed/mbedtls/pull/1919/files#r246046772, please clarify what “ciphers” covers.